ign_rviz_plugins/src/rviz/plugins/GlobalOptions.cpp

```diff
@@ -104,6 +145,9 @@ bool GlobalOptions::eventFilter(QObject * _object, QEvent * _event)
     this->onRefresh();
   }
 
+  // Update tf status and message
+  this->updateTfStatus();
```

Comment:
Currently, the `updateTfStatus()` function is being called on every Qt event received by this plugin. This results in a high TF status refresh rate which I don't think is necessary. This might also cause problems with other plugins using `FrameManager` due to the presence of locks.

I am not 100% sure on how to go about this, but updating the TF status alongside the periodic refresh of TF frames might be the right option. The periodic update of TF is currently under development in #62.